docs/pages/Configuration.md

```diff
@@ -96,25 +96,144 @@ However, it is recommended that modules include their namespace in the virtual p
 resources:
   configuration:
     paths:
-      "/resources/my/library/module-xy/": lib
-      "/resources/my/library/module-xy-min/": dist
+      /resources/my/library/module-xy/: lib
+      /resources/my/library/module-xy-min/: dist
 ````
 
 ### Encoding of `*.properties` files
-*Available since UI5 CLI [`v1.7.0`](https://github.com/SAP/ui5-cli/releases/tag/v1.7.0)*
 
-By default, the UI5 Tooling expects `*.properties` files to be `ISO-8859-1` encoded. 
+!!! info
+    This configuration is available since UI5 CLI [`v1.7.0`](https://github.com/SAP/ui5-cli/releases/tag/v1.7.0)
 
-If your project uses `UTF-8` encoding for these files (for example because it has been created in SAP WebIDE), you need to set the `propertiesFileSourceEncoding` configuration property.
+By default the UI5 Tooling expects different encodings for `*.properties` i18n files, depending on a projects specification version:
 
-Your projects `*.properties` files will be read in the given encoding and any non-ASCII characters replaced with the respective unicode escape sequences. This allows you to deploy the resulting files to any environment, independent from how it expects `*.properties` files to be encoded. Please refer to [RFC 7](https://github.com/SAP/ui5-tooling/blob/master/rfcs/0007-properties-file-encoding.md) for details.
+Specification Version | Default propertiesFileSourceEncoding
+--- | ---
+**0.1, 1.0 or 1.1** | `ISO-8859-1` 
+**2.0+** | `UTF-8`
+
+If your project uses a different encoding for `*.properties` files, you need to set the `propertiesFileSourceEncoding` configuration property.
+
+The UI5 Tooling will read the corresponding files of the project in the given encoding. Any non-ASCII characters will be replaced with the respective Unicode escape sequences. This allows you to deploy the resulting files to any environment, independent from how it expects `*.properties` files to be encoded. Please refer to [RFC 7](https://github.com/SAP/ui5-tooling/blob/master/rfcs/0007-properties-file-encoding.md) for details.
 
 ````yaml
 resources:
   configuration:
-    propertiesFileSourceEncoding: UTF-8
+    propertiesFileSourceEncoding: UTF-8|ISO-8859-1
 ````
 
+## Framework Configuration
+
+!!! info
+    This configuration is available since UI5 CLI [`v2.0.0`](https://github.com/SAP/ui5-cli/releases/tag/v2.0.0)
+    and applies only to projects defining [Specification Version](#specification-versions)
+    2.0 or higher.
+
+Define your projects framework dependencies.
+
+Your projects framework configuration you must define whether you want to use the OpenUI5 or the SAPUI5 framework. Also see our [documentation on the differences between OpenUI5 and SAPUI5](./SAPUI5.md#differences-between-openui5-and-sapui5).
```

Comment:
Thats nice, but the page does not exist at the moment. Instead of creating another documentation we could use (and extend):
https://sapui5.hana.ondemand.com/#/topic/5982a9734748474aa8d4af9c3d8f31c0

Reply:
This PR adds that page: https://github.com/SAP/ui5-tooling/pull/221/files#diff-f04285aac548e037b20b45383c4c8ed3R60

And also links to the demokit: https://github.com/SAP/ui5-tooling/pull/221/files#diff-f04285aac548e037b20b45383c4c8ed3R68

But yes, we could remove that chapter here and just link to the demokit...